content/docs/started/_index.md

```diff
@@ -3,3 +3,25 @@ title = "Getting Started"
 description = "How to get started using Kubeflow."
 weight = 20
 +++
+
+The guides in this section give detailed information about using Kubeflow and 
+its components.
+
+For best understanding of the guides, it's useful to have some knowledge of
+the following systems:
+
+* [Kubernetes](https://kubernetes.io/docs/tutorials/kubernetes-basics/)
+* [TensorFlow](https://www.tensorflow.org/get_started/)
+* [ksonnet](https://ksonnet.io/docs/tutorial)
```

Comment:
Is `ksonnet` still a requirement?

Reply:
It's not, but I wanted to focus this PR on moving the existing docs around instead of making sure all the content in them is up to date. I believe there are some others working on removing ksonnet from the docs, and I don't want to duplicate their efforts

Reply:
The removal of ksonnet from this page has already happened:
https://www.kubeflow.org/docs/started/requirements/
The above page is the source of the content that this PR moves to a different page. This is one of the reasons for the merge conflict that GitHub is reporting. Please would you do a careful sync and merge. :)